tensorflow/core/kernels/BUILD

```diff
@@ -8224,7 +8224,14 @@ tf_mkl_kernel_library(
 tf_mkl_kernel_library(
     name = "mkl_fused_batch_norm_op",
     srcs = ["mkl_fused_batch_norm_op.cc"],
-    deps = NN_DEPS + mkl_deps(),
+    hdrs = [
+        "fused_batch_norm_op.h",
+        "no_op.h",
+    ],
```

Comment:
I don't think you need these. `hdrs` is [a list of header files published by this library](https://docs.bazel.build/versions/master/be/c-cpp.html#cc_library.hdrs) (`mkl_fused_batch_norm_op`). By including `:fused_batch_norm_op` you should already have visibility to its `fused_batch_norm_op.h`.

Reply:
Done